pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -659,11 +659,19 @@ func (a *ApplicationRestoreController) restoreVolumes(restore *storkapi.Applicat
 		logrus.Errorf("error in checking backuplocation type")
 		return err
 	}
+	// Create a flag for onlyPXDdriver so that we don't create pvc job for pxd driver
+	var onlyPXDdriver bool
+	if len(backupVolumeInfoMappings) == 1 {
+		_, onlyPXDdriver = backupVolumeInfoMappings[volume.PortworxDriverName]
```

Comment:
Can you please comment in what conditions this will happen? Is it when there is no other volumeMapping other than Pxd?